package-lock.json

```diff
@@ -0,0 +1,6485 @@
+{
+  "name": "xzbfsf-stro",
+  "version": "0.0.1",
+  "lockfileVersion": 3,
```

Comment:
in keeping with semantic versioning, since not shipped yet, incorrect to change version to 1.0.0 :(( i hate 0.0.01 but unfortunately we'll keep it for now.